source/user-guide/associating-resources/cite.rst

```diff
@@ -0,0 +1,62 @@
+.. _cite:
+
+Cite a resource
+###############
+
+The catalogue can provide an automatic citation based on information provided in the metadata record.
+
+.. figure:: img/citation.png
+
+The citation can be use to cite a dataset in publications.
+
+
+Citation text
+-------------
+
+The citation generated by default follows general conventions on how to cite a dataset but can be customized.
+
+The default is:
+* list of authors (and organisation)
+* publication year
+* title
+* list of publisher
+* DOI URL or landing page URL.
+
+Those information are the one required to create DOI (see :ref:`doi`).
+
+Customization of the citation can be done in :code:`schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/formatter/citation/common.xsl`.
+
+An additional text may be stored in an online resource description with protocol ``WWW:LINK-1.0-http--publication-URL``.
+
+.. figure:: img/citation-with-addition.png
+
+Example: https://sextant.ifremer.fr/Donnees/Catalogue#/metadata/3f8eaace-9f9b-4b1b-a7a4-9c55270e205a
```

Comment:
I would remove the example URL, as it can change or can be not available in the future.

Reply:
Those records have DOI so they are supposed to continue to exist ;) Maybe I can replace the link by the DOI https://doi.org/10.12770/ad07a55f-5de7-4abc-ba89-8899b16c4b59

Putting example of real catalogues in the doc is a good way to illustrate what can be done with GeoNetwork and closer link to end users I think - Maybe we can better check links we put in the doc but I would not remove them.